checkov/kubernetes/checks/Seccomp.py

```diff
@@ -57,10 +57,8 @@ def scan_spec_conf(self, conf):
                             if "metadata" in conf["spec"]["jobTemplate"]["spec"]["template"]:
                                 metadata = conf["spec"]["jobTemplate"]["spec"]["template"]["metadata"]
         else:
-            if "spec" in conf:
-                if "template" in conf["spec"]:
-                    if "metadata" in conf["spec"]["template"]:
-                        metadata = conf["spec"]["template"]["metadata"]
+            if conf.get("spec") and conf.get("spec").get("template"):
+                metadata = conf.get("spec").get("template").get("metadata", {})
```

Comment:
@rotemavni Can we leverage the new `get_inner_spec`?

Reply:
@nimrodkor Good idea, renamed `get_inner_conf` to be `get_inner_entry` and added the entry name as param